api/pkg/apis/v1alpha1/model/campaign.go

```diff
@@ -125,9 +125,11 @@ func (c ActivationState) DeepEquals(other IDeepEquals) (bool, error) {
 }
 
 type CampaignSpec struct {
-	FirstStage  string               `json:"firstStage,omitempty"`
-	Stages      map[string]StageSpec `json:"stages,omitempty"`
-	SelfDriving bool                 `json:"selfDriving,omitempty"`
+	FirstStage   string               `json:"firstStage,omitempty"`
+	Stages       map[string]StageSpec `json:"stages,omitempty"`
+	SelfDriving  bool                 `json:"selfDriving,omitempty"`
+	Version      string               `json:"version,omitempty"`
```

Comment:
Is it better to put Version and RootResource in ObjectMeta?

Reply:
Version and RootResource are used to add labels. The properties are mapped using liquid template in k8s bridge.

Reply:
Usage of these properties will be shown in next PR for changes in api/controllers.